_data/CONTRIBUTORS.yaml

```diff
@@ -540,6 +540,12 @@ Dirk Winkelhardt:
     orcid: 0000-0001-8770-2221
     email: dirk.winkelhardt@rub.de
     affiliation: Ruhr-Universität Bochum, Medzinisches Proteom-Center & ELIXIR Germany Forschungszentrum Jülich
+Carolina Simón Guerrero:
+    git: csimong
+    email: carolina.simon@cnb.csic.es
+    orcid: 0000-0002-0735-4317
+    role: editor
```

Comment:
@csimong I do not think you have the "role" of an "editor" (although you can become one if you want to).
I would suggest getting rid of that line for now.

Reply:
No problem! I will delete it.